hack/verify-generated.sh

```diff
@@ -60,8 +59,7 @@ main() {
   diff=$(diff -Nupr \
           -x ".git" \
           -x "_output" \
-          -x "vendor/github.com/jteeuwen/go-bindata/testdata" \
-          -x "vendor/github.com/golang/dep/internal/fs/testdata/symlinks" \
+          -x "vendor" \
```

Comment:
this one still needs to ignore vendor because hack/update-generated.sh locally populates vendor (because the code-generators currently need to be run in GOPATH)